css/wsuwp-plugin-idonate.css

```diff
@@ -730,6 +730,18 @@
     font-size: 16px;
   }
 }
+#fundSelectionForm .help-icon a#moreAdvFeeInfo {
+  display: inline-block;
+  right: 0;
+  top: 0;
+  width: 15px;
+  height: 15px;
+  padding: 0;
+  background: url(../images/ico-help.png) no-repeat center center;
+  background-size: 15px;
```

Comment:
If we are going to use a 15px image, we should have a resized version of the icon as well. The current icon doesn't scale very well in IE at at all. 
![fund030](https://user-images.githubusercontent.com/11821485/27664913-f3e15f08-5c1e-11e7-81b1-c1f9049a4164.png)